addons/product_configurator/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': "Product Configurator",
+    'version': '1.0',
+    'category': 'Hidden',
+    'summary': "Configure your products",
+
+    'description': """
+Technical module:
+The main purpose is to define the basics of the product configuration in forms.
+
+It also enables the "optional products" feature.
+    """,
+    'depends': ['product'],
+    'assets': {
+        'web.assets_backend': [
+            'product_configurator/static/src/**/*',
+        ],
+    },
+    'auto_install': True,
```

Comment:
I'm not sure about the auto_install. Maybe we should only install it when it's required by another module